### DISCUSSION TITLE
Multiple entry points

### DISCUSSION THREAD

**Discussion by hywax** [nuxt/nuxt#27375] [Questions]
Is there an option to add an additional entry point?

Project structure
```
├── ...
└── server
    ├── ...
    └── worker.ts
```

I expect to have a `worker.js` file in the `.output/server/worker.js` path when I build it. To then run it as a separate process.

I tried adding to `vite` config, but the file goes to `public` on build

```
vite: {
  build: {
    rollupOptions: {
      input: {
        worker: path.resolve(__dirname, 'server/worker.ts'),
      },
    },
  }
},
```

> About my case
> I need queues in the project, and since Nuxt doesn't have this, I would like to make a separate worker that uses the same imports/types and lies in the project folder

**Comment by HigherOrderLogic** [comment]
Have you take a look at [Nitro Task](https://nitro.unjs.io/guide/tasks)?

**Reply by hywax** [reply]
Tasks are started in the same process. This is unsafe and also overloads the underlying process. 

I want to run a separate worker that will handle job

**Reply by qodeboy** [reply]
@hywax have you managed to find a workaround?

**Comment by karladler** [comment]
I would need that feature too, to bundle workers loaded by third party modules.
Currently I see no way to actually exclude certain files from external node module. so it stays in the .output/node_modules for later dynamic imports.

**Comment by julbd** [comment]
@hywax Did you found a solution to use workers with dependencies ?
